platform/ui/src/components/userPreferencesModal/AboutModal.js

```diff
@@ -1,31 +1,17 @@
-import React, { Component } from 'react';
-import PropTypes from 'prop-types';
-import Modal from 'react-bootstrap-modal';
+import React from 'react';
```

Comment:
`AboutModal.js` feels like an odd component to have as reusable.
Can we push it up a directory? `ui/src/components/AboutModal/`